src/export/export_snapshot_s3_function.py

```diff
@@ -0,0 +1,52 @@
+import os
+import time
+
+import boto3
+
+import constants
+import utility as util
+
+
+def lambda_export_rds_snapshot_to_s3(event, context):
+    """start export task of RDS snapshot to S3 bucket"""
+    region = os.environ['Region']
+    rds = boto3.client('rds', region)
+    result = {}
+    instance_id = event['identifier']
+    epoch = int(time.time())
+    export_id = instance_id + "-" + str(epoch)
+    snapshot_id = instance_id + constants.SNAPSHOT_POSTFIX
+    snapshot_arn = get_instance_snapshot_arn(snapshot_id)
+    account_id = util.get_aws_account_id()
+    bucket_name = constants.RDS_SNAPSHOTS_BUCKET_NAME_PREFIX + account_id
```

Comment:
@RiyaJohn is it assumed that the s3 bucket is already existing? and if it doesnt, the export task throws an exception?

Reply:
Bucket will exist as the bucket is created as part of our deployment here https://github.com/intuit/Trapheus/blob/4d228832296da832e7947003027768c025dbafb0/template.yaml#L370. Don't remember if it will throw error or will create one, will test and update here. Also yes will add a test case for it

Reply:
Doesn't throw an exception, when the bucket doesn't exist. Just creates the bucket, which imo is correct (better).

Reply:
@despot agreed. If its being created as part of the deployment, i think we are ok. But considering a scenario that the s3 bucket can be deleted, do you think its good to have that handled?

Reply:
@namitad thanks for considering my opinion. I was trying to find some articles about what i already practiced within the team in the past, though I would need to dig deeper. The idea is, that usually the application refers to doing one thing, and as soon as the team starts handling cases (including only exception handling) what a user can do outside of the prescribed way, the team goes into the direction of solving the problem of the universe. More pragmatically, the team wouldn't wanna handle someone deleting a lambda function or the kmsKeyId manually e.t.c. as this is not what they should do. So I don't think a test should be written for the user deleting the S3 bucket. It all comes down to what you want the application to do. Does a user deleting a S3 bucket a normal behavior that you expect to handle. I don't imagine this app like that. But if you do, then either handle the case (by, for instance, creating a new S3 bucket) or have a custom exception so you remember to do this in future. As I think this shouldn't be handled, imo @RiyaJohn should not do anything more regarding the S3 bucket. But again, you are leading the project :), so let us know.

Reply:
Totally agree with @despot. But already pushed code for this test case, so @namitad you can take a look at it and let me know https://github.com/intuit/Trapheus/pull/5/commits/b4701c8914f78ea1fbd2bbf275a3f569a04c1992